assets/xml/textures/place_title_cards.xml

```diff
@@ -0,0 +1,287 @@
+<Root>
+    <File Name="g_pn_01">
+        <Texture Name="gForestTempleTitleCardEngTex" OutName="forest_temple_english" Format="ia8" Width="144" Height="24" Offset="0x0"/>
+        <Texture Name="gForestTempleTitleCardGerTex" OutName="forest_temple_german" Format="ia8" Width="144" Height="24" Offset="0xD80"/>
+        <Texture Name="gForestTempleTitleCardFraTex" OutName="forest_temple_french" Format="ia8" Width="144" Height="24" Offset="0x1B00"/>
```

Comment:
I think we should try to be consistent when naming for different languages, and it looks like so far we've been using `_eng`/`_ger`/`_fra` for file names + `ENG`/`GER`/`FRA` for symbol names

Reply:
agreed